packages/common/src/core/types/IHubInitiative.ts

```diff
@@ -11,6 +12,8 @@ export interface IHubInitiative
     IWithSlug,
     IWithCatalog,
     IWithMetrics,
-    IWithPermissions {}
+    IWithPermissions {
+  status: HubEntityStatus;
+}
```

Comment:
I think we might want to have an `IWithStatus` which exposes `{ status: HubEntityStatus; }`. Thoughts @dbouwman ?

Reply:
Agreed - good catch @benstoltz :)